plugin/jupycent.vim

```diff
@@ -107,7 +107,7 @@ function! s:cleanup()  "{{{
 endfunction  "}}}
 
 function! JupycentFold(lnum)  "{{{
-  let l:line = getline(a:lnum) 
```

Comment:
I think you accidentally added a trailing space.

Reply:
Hi @gabenespoli that's I removed a trailing space. :)

Reply:
Haha oops, my bad!